lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -515,8 +517,9 @@ fn test_monitor_update_fail_cs() {
 			assert_eq!(*node_id, nodes[0].node.get_our_node_id());
 
 			*nodes[0].chan_monitor.update_ret.lock().unwrap() = Err(ChannelMonitorUpdateErr::TemporaryFailure);
-			if let msgs::LightningError { err, action: msgs::ErrorAction::IgnoreError } = nodes[0].node.handle_commitment_signed(&nodes[1].node.get_our_node_id(), &updates.commitment_signed).unwrap_err() {
-				assert_eq!(err, "Failed to update ChannelMonitor");
+			nodes[0].node.handle_commitment_signed(&nodes[1].node.get_our_node_id(), &updates.commitment_signed);
+			if let MessageSendEvent::HandleError { .. } = nodes[0].node.get_and_clear_pending_msg_events()[0] {
```

Comment:
Same here as above - why not keep the IgnoreError and message-in-HandleError check that was here (even if you're also gonna add the log message tests)?